locations/rauchhaus.yml

```diff
@@ -55,6 +55,13 @@ networks:
     ipv6_subprefix: -1
     ptp: true
 
+  - vid: 11
+    role: mesh
+    name: mesh_xdorf
+    prefix: 10.31.98.189/32
```

Comment:
This IP might not be assigned to this locations. Please check the reserved IPs

Reply:
I already asked the owner. Waiting for a reply.